app/scripts/services/aggregatedLoggingService.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("AggregatedLoggingService", function($q, $cacheFactory, Logger, $interval, APIService, DataService) {
+
+    var cachedUserPermissions = $cacheFactory('operationsUsersCache', {
+      number: 10
+    });
+
+    var inFlightPermissionsRequest = null;
```

Comment:
I don't think this is needed since you should never have multiple in flight requests for this specific permission